plugins/english/wtrlab.ts

```diff
@@ -0,0 +1,289 @@
+import { Plugin } from "@typings/plugin";
+import { fetchApi, fetchFile } from "@libs/fetch";
+import { FilterTypes, Filters } from "@libs/filterInputs";
+import { load as parseHTML } from "cheerio";
+
+class WTRLAB implements Plugin.PluginBase {
+  id = "WTRLAB";
+  name = "WTR-LAB";
+  site = "https://wtr-lab.com/";
+  version = "1.0.0";
+  icon = "src/en/wtrlab/icon.png";
+  lang = "en/"
```

Comment:
```suggestion
```

Reply:
it's necessary because it's multi-lang source. https://github.com/LNReader/lnreader-sources/pull/797/commits/6620dd8901611bae1e67223d1a7f8432c16bceb3

Reply:
ID source request issues https://github.com/LNReader/lnreader-sources/issues/714

Reply:
so you should use another variable name. `lang` is used by app

Reply:
Done